Gravity/Gravity/DAL/SQL/SqlDao.Get.cs

```diff
@@ -191,38 +190,26 @@ private object GetChildObjectRecursively(Dictionary<Guid, string> fieldsGuidsToC
 
 		private Dictionary<Guid, string> GetArtifactGuidsMappingsToColumnNames(IEnumerable<Guid> guids)
 		{
-			var returnDictionary = new Dictionary<Guid, string>();
-
 			StringBuilder sqlStringBuilder = new StringBuilder(SQLGetResource.GetArtifactGuidsMappingToColumnNames);
 
 			sqlStringBuilder.Replace("%%ArtifactGuids%%", string.Join(",", guids.Select(g => $"'{g}'")).TrimEnd(','));
 
 			DataTable dtTable = dbContext.ExecuteSqlStatementAsDataTable(sqlStringBuilder.ToString());
 
-			if (dtTable.Rows.Count > 0)
-			{
-				returnDictionary = dtTable.AsEnumerable().ToDictionary(row => (Guid)row[0], row => (string)row[1]);
-			}
-
-			return returnDictionary;
+			return dtTable.Rows.Count > 0 ?
```

Comment:
AsEnumerable() will work even when it is empty. But can can be cleaned up another time :smiley: